components/comment/video/like-widget/like-widget.ts

```diff
@@ -43,7 +43,7 @@ export class AppCommentVideoLikeWidget extends Vue {
 			return false;
 		} else if (this.video.comment.user.id === this.app.user.id) {
 			return false;
-		} else if (this.video.game.developer.id === this.app.user.id) {
+		} else if (this.video.game.hasPerms()) {
```

Comment:
i think we should still let collaborators vote, otherwise old votes will not be togglable